sandbox/handlers/utils.js

```diff
@@ -47,8 +47,5 @@ export function hasValidGlobalTemplatePersonalisation(personalisation) {
     return false;
   }
 
-  if (personalisationFields[0] !== "body") {
-    return false;
-  }
-  return true;
+  return personalisationFields[0] == "body";
```

Comment:
```suggestion
  return personalisationFields[0] === "body";
```

Minor and complete edge case but this would return `true` if personalisationFields[0] equalled `true` when using `==`, love javascript

Reply:
Yep thanks for that, I come from java and `===` still confuses me a lot :D